.gitignore

```diff
@@ -0,0 +1,4 @@
+.mc656/*
+.mc656/
```

Comment:
This was a python virtual environment that I created to test the code. Subject to change